agents/ocf/HealthCPU.in

```diff
@@ -205,9 +205,9 @@ case "$__OCF_ACTION" in
 meta-data)      meta_data
                 exit $OCF_SUCCESS
                 ;;
-start)          healthcpu_start;;
+start)          healthcpu_validate; healthcpu_start;;
 stop)           healthcpu_stop;;
-monitor)        healthcpu_monitor;;
+monitor)        healthcpu_validate; healthcpu_monitor;;
```

Comment:
Rather than exit from validate (which means we wouldn't get the debug log at the end) you can do `healthcpu_validate && healthcpu_monitor` here

Reply:
That's a good idea!